src/ibverbs/device_context.rs

```diff
@@ -45,6 +68,212 @@ impl From<u32> for Mtu {
     }
 }
 
+#[repr(u8)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
+pub enum PortWidth {
```

Comment:
We can remap the value of `PortWidth` to the natural one (e.g. `Width2X = 2`, `Width12X = 12`, etc.) so that `to_multiplier()` do not have to enumerate all the possibilities.